src/css/styles/_c-form.scss

```diff
@@ -0,0 +1,177 @@
+.c-form {
```

Comment:
IMHO it looks better when it has margins

![image](https://user-images.githubusercontent.com/789841/46853727-34bdab00-cdff-11e8-8f62-b6215e59db5b.png)

vs

![image](https://user-images.githubusercontent.com/789841/46853796-6f274800-cdff-11e8-8961-f809b51d1961.png)

- [ ] margins